components/twilio/actions/list-messages/list-messages.mjs

```diff
@@ -42,14 +42,23 @@ export default {
     if (this.to) {
       const toParsed = phone(this.to);
       if (!toParsed || !toParsed.phoneNumber) {
-        throw new Error(`Phone number ${this.to} couldn't be parsed as a valid number.`);
+        throw new Error(`Phone number ${this.to} could not be parsed as a valid number.`);
       }
       to = toParsed.phoneNumber;
     }
 
+    let from = this.from;
+    if (this.from) {
+      const fromParsed = phone(this.from);
+      if (!fromParsed || !fromParsed.phoneNumber) {
+        throw new Error(`Phone number ${this.from} could not be parsed as a valid number.`);
```

Comment:
```suggestion
        throw new Error(`Phone number ${this.from} couldn't be parsed as a valid number.`);
```

Reply:
Changed 'could not' to 'couldn't' for conciseness (Google's Developer docs style guide)